### ISSUE TITLE
Multimodal Contrastive Learning with LIMoE: the Language-Image Mixture of Experts

### ISSUE THREAD

**Issue by reyllama** [reyllama/paper-reviews#176]
# TL; DR

- Single Tower CLIP w/ Mixture of Experts (MoE) modules
- Contrastive Learning + Entropy Regularizations to fully utilize experts with confident partitioning
- Near SOTA image recognition and image-text retrieval performances + Favorable compute-yield trade-off

# Motivation

- Sparse MoE: (1) compute efficiency (2) continual learning (3) multi-task learning
- Single Tower Multimodal learner: (1) Generality (2) Scalability (3) Cross-modal / Cross-task transferability

# Formulation

<img width="445" alt="image" src="https://user-images.githubusercontent.com/46308407/176618747-97f1d0c6-7739-46ff-b444-278ddf0d7e78.png">

### Architecture

- Modality-aware token embeddings
- Modality-agnostic transformer blocks
- Modality-aware linear projectors

# Evaluation

# Inspiration
